opaque_keys/edx/keys.py

```diff
@@ -77,6 +77,26 @@ def block_type(self):  # pragma: no cover
         raise NotImplementedError()
 
 
+class AggregateCourseKey(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a
+    serialized Course Key object.
+    """
+    KEY_TYPE = 'aggregate_course_key'
```

Comment:
@cpennington @clintonb  I have to change the key type to `course_key_v2` or just `course_v2` because the `CANONICAL_NAMESPACE` of `AggregateCourseLocator` would be `course-v2`? Also should I change the name of aggregate course key and locator names too?